src/Initializer/DynamicRupture.h

```diff
@@ -127,8 +131,11 @@ struct seissol::initializers::DynamicRupture {
     tree.addVar(         waveSpeedsMinus,             mask,                 1,      MEMKIND_STANDARD );
     tree.addVar(          drEnergyOutput,             mask,         ALIGNMENT,      MEMKIND_STANDARD );
     tree.addVar(      impAndEta,                      mask,                 1,      MEMKIND_STANDARD );
+    tree.addVar(      impedanceMatrices,              mask,                 1,      MEMKIND_STANDARD );
```

Comment:
No, we don't always need them, but the overall agreement is to reduce ifdefs wherever possible.
If I add an ifdef here, we would need more ifdefs in CellLocalMatrices.cpp
Otherwise I could create a more complicated class structure with a poroelastic DR LTS tree and a non-poroelastic DR LTS tree, which I would like to avoid. If we add the matrices here and just leave them in the tree, they don't do any harm, IMHO.